.changeset/lucky-flies-search.md

```diff
@@ -0,0 +1,4 @@
+---
+---
+
+chore: release deprecation script
```

Comment:
This has been tested as far as seeing the correct versions and failing the deprecate command, but needs to be run with the correct perms (in CLI) to do a full test.

Reply:
@danielbate Can't our CI user run this?

Reply:
It can, but I was awaiting approvals before I tried a run in deprecation mode 👍🏻